### PR TITLE
[AC-1487] Fix missing Admin Auth Request emails

### PR DESCRIPTION
**Pull request by shane-melton**
## Type of change

```
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective
Update the TDE admin auth request SQL queries to use the `[User]` table for the email address instead of the `[OrganizationUser]` table, which does not always have an email available.

## Code changes

Update the `AuthRequest_ReadAdminApprovalsbyIds.sql` and `AuthRequest_ReadPendingByOrganizationId.sql` scripts to join on the `[User]` table in order to retrieve the user's email address.

Also adds the necessary migration script.

## Before you submit

- Please check for formatting errors (`dotnet format --verify-no-changes`) (required)
- If making database changes - make sure you also update Entity Framework queries and/or migrations
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

**Review by justindbaur — APPROVED** [review]
Looks great, thank you!

File: src/Sql/Auth/dbo/Stored Procedures/AuthRequest_ReadPendingByOrganizationId.sql [hunk 1]
```diff
@@ -5,11 +5,13 @@ BEGIN
     SET NOCOUNT ON
 
 SELECT
-    ar.*, ou.[Email], ou.[OrganizationId], ou.[Id] AS [OrganizationUserId]
```
[the remark this answers is not in the document]
Comment (shane-melton) [reply src/Sql/Auth/dbo/Stored Procedures/AuthRequest_ReadPendingByOrganizationId.sql:8]: Yep, it's already part of the `AuthRequest` table and was redundant with the `ar.*` selection.

[the remark this answers is not in the document]
Comment (JaredSnider-Bitwarden) [reply src/Sql/Auth/dbo/Stored Procedures/AuthRequest_ReadPendingByOrganizationId.sql:8]: Ah, perfect. Just wanted to make sure!

**Review by JaredSnider-Bitwarden — APPROVED** [review]
LGTM! Thank you!

**Comment by vincentsalucci** [comment]
Removing the `needs-qa` tag for merging into `master` and pulling in latest changes for further QA testing on `feature/trusted-device-encryption`

**Pull request merged**